docs/fast_data/tutorials/low-code.mdx

```diff
@@ -588,23 +631,93 @@ Here's the ER Schema we used before, which must be pasted inside the Editor of t
 }
 ```
 
-For the aggregation, we will use the relationship from _registry_ to _order_, since we will aggregate the Single View starting from the document found in _registry_ that will include the list of
-_order_ accessed via the condition _registry_to_order_.
+In this example, we can see that the relationship goes from registry to order (from line 4 to line 16) and vice versa.
+
+The relationship is represented by the following two conditions:
+
+- `registry_id` (from `orders`): `registry_id` (from the updated registry). The condition is `One to One`, meaning that one order can be related to one and only one registry item.
+- `registry_id` (from `registry`): `registry_id` (from the updated order). The condition is `One to Many`, meaning that one registry might have a relation with multiple orders.
+
+The conditions will be interpolated into the `find` operation of a MongoDB query. In this case, the condition can be translated as:
+
+- `registry_to_order`: `find({ registry_id: "value of registry_id from registry projection"})`
+- `order_to_registry`: `findOne({ registry_id: "value of registry_id from orders projection"})`
+
+</TabItem>
+
+
+</Tabs>
+
+### Single View Key
+
+The **Single View Key** represents the link between the identifier of the projection change and the Single View fields: this mapping will be executed at the beginning of the aggregation, to retrieve the **base projection** that will be used as a starting point to compute the single view.
+to get a query that, executed in the Single View collection, retrieves the Single View document to be updated.
+
+<Tabs
+  defaultValue="no-code"
+  groupId="fast-data-mode"
+  values={[
+      { label: 'No Code', value: "no-code", },
+      { label: 'Low Code', value: "low-code", },
+  ]}
+>
+
+
+<TabItem value="no-code">
+
+:::info
+The following explains how to configure a Single View Key using the "No Code" feature, available from version `11.3.0` of the Mia-Platform Console.
+Please make sure that your project has activated this feature. You might need to ask your Project Administrator for it.
+:::
+
+To configure the _Single View Key_ with the "No Code" feature, you can refer to the [specific documentation page](/fast_data/configuration/config_maps/singleViewKey.md#using-the-no-code).
+
+Here is a quick list of operations to execute to configure correctly the Single View Key:
+
+* move to the **Settings** page and go to the **Single View Key** card.
```

Comment:
here, before it is necessary to set the base projection, but I don't find this step. In fact, at this step, when I go to Settings -> Single View Key card, I cannot see anything to do, only a messagge: 'You need to set the base projection from the general settings to configure the Single View Key'

Reply:
since it's an important concept, I've added a paragraph about it after er schema